spec/classes/puppet_server_config_spec.rb

```diff
@@ -201,7 +201,6 @@
 
         it 'should contain autosign.conf with out content set' do
           should contain_file("#{confdir}/autosign.conf")
-          should_not contain_file("#{confdir}/autosign.conf").with_content(/# Managed by Puppet/)
```

Comment:
Could you keep this in but modified to the current header?

Reply:
Done